source/reference/deprecated/mc-ilm-add.rst

```diff
@@ -122,7 +122,7 @@ Parameters
    For versioned buckets, the expiry rule applies only to the *current* object version. 
    Use the :mc-cmd:`~mc ilm add --noncurrent-expire-days` option to apply expiration behavior to noncurrent object versions.
 
-   MinIO uses a scanner process to check objects against all configured lifecycle management rules. 
+   MinIO uses a :ref:`scanner process <minio-concepts-scanner>` to check objects against all configured lifecycle management rules. 
```

Comment:
Ok it is indeed `all` but `the` reads more smoothly for me here? Use if you like.
```suggestion
   MinIO uses a :ref:`scanner process <minio-concepts-scanner>` to check objects against the configured lifecycle management rules. 
```

Reply:
oh and there's a bunch of these

Reply:
I'm not for or against this change. I'll take it, though.